lib/utils.pm

```diff
@@ -402,6 +403,21 @@ sub zypper_call {
     return $ret;
 }
 
+sub zypper_enable_install_dvd {
+    # If DVD Packages is used we need to (re-)enable the local repos
+    # see FATE#325541
+    zypper_call 'mr -e -l' if get_var('AUTOYAST')
+      || is_sle('15+') and get_var('ISO_1', '') =~ /SLE-.*-Packages-.*\.iso/;
+    zypper_call 'ref';
```

Comment:
I take that "Enable dvd's" it's a step on it's own, and that includes the ref. Should we rename or repurpose the zypper_ref?

Reply:
Actually you can skip `AUTOYAST` variable check, as it also has `ISO_1` (all packages DVD mounted).

Reply:
@rwx788 dropped